assets/js/components/settings/SettingsStatuses.js

```diff
@@ -21,11 +21,30 @@
  */
 import PropTypes from 'prop-types';
 
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ProgressBar } from 'googlesitekit-components';
+
 export default function SettingsStatuses( { statuses } ) {
 	if ( ! statuses || statuses.length === 0 ) {
 		return null;
 	}
 
+	function renderStatus( status ) {
+		if ( status === undefined ) {
+			return <ProgressBar />;
```

Comment:
There are console warnings for nesting `div` within the `p` tag.

![Screenshot 2024-12-10 at 2 07 21 PM](https://github.com/user-attachments/assets/3ca20976-992e-4755-990c-aa52afe4dacd)